### PR TITLE
[ON HOLD] Households Data

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

Add the number and size of households rural vs urban
### To Do
- [ ] Tanzania
- [x] Senegal
- [ ] Nigeria

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 python manage.py migrate --noinput        # Apply database migrations
 cat takwimu/sql/*.sql | psql              # Upload tables / data
+cat takwimu_dev.sql | psql                # Upload test data
```
Comment (DavidLemayian) [inline docker-entrypoint.sh]: @phillipahereza This will also happen in production...

File: takwimu/static/css/takwimu.css [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+  */ }
+
+/*
+* Extra large devices (large desktops, 1200px and up)
+*/
+@media (min-width: 1200px) {
+  /*
+  * Do not delete the code below, it's useful for css debuging
+  * during development
+  */ }
```
Comment (DavidLemayian) [inline takwimu/static/css/takwimu.css]: @phillipahereza Are you using docker-compose? My setup doesn't create this file.. 🤔

**Comment by phillipahereza** [comment]
@DavidLemayian sorry, I was meant to remove that

File: takwimu/static/css/takwimu.css (hunk 2 above)
Comment (phillipahereza) [reply takwimu/static/css/takwimu.css]: @DavidLemayian sorry, I meant to remove that after populating the DB

**Comment by kilemensi** [comment]
Ping @phillipahereza, seems like Sedhiou (SN_1_011) is missing from the data.

**Comment by DavidLemayian** [comment]
@kilemensi Why is this "On Hold"?

**Comment by kilemensi** [comment]
Sedhiou (SN_1_011) is still missing from the data @DavidLemayian. cc @phillipahereza

**Comment by DavidLemayian** [comment]
👀

**Comment by kilemensi** [comment]
Takwimu data team are revising all national/subnational data and we'll rely on their input moving forward.

**Pull request closed without merging**